DigitalLearningSolutions.Data/Services/DelegateUploadFileService.cs

```diff
@@ -195,7 +195,7 @@ private void UpdateDelegate(DelegateTableRow delegateRow, DelegateUser delegateU
         private void RegisterDelegate(DelegateTableRow delegateRow, DateTime? welcomeEmailDate, int centreId)
         {
             var model = new DelegateRegistrationModel(delegateRow, centreId, welcomeEmailDate);
-            var errorCodeOrCandidateNumber = registrationDataService.RegisterDelegate(model);
+            var errorCodeOrCandidateNumber = registrationDataService.RegisterNewUserAndDelegateAccount(model);

```

Comment:
RegisterNewUserAndDelegateAccount() no longer returns an error code. We should make sure we update any code that calls it and ensure that the failure scenarios are dealt with - can you leave a TODO for [HEEDLS-886](https://softwiretech.atlassian.net/browse/HEEDLS-886) so that that gets done?

Reply:
The only failure codes are generic failure and duplicate emails, but yes, this needs to be handled clearly.